go/cli/mcap/README.md

```diff
@@ -38,6 +38,8 @@ To install from the latest commit, use
 
     go install github.com/foxglove/mcap/go/cli/mcap@latest
 
+Ensure that the go installation directory is in your path (e.g. `$HOME/go/bin` by default on Ubuntu).
```

Comment:
Actually we should probably remove this section (see https://github.com/foxglove/mcap/pull/826/files) -- cc @wkalt

Reply:
Hmm we already state that it is not recommended, but I think we should leave it in as it's helpful for dev.

Reply:
I can add a (not recommended) to make it obvious